sphinxcontrib/adadomain.py

```diff
@@ -25,6 +25,9 @@
 from sphinx.util.docfields import Field, TypedField
 from sphinx.util.nodes import make_refnode, make_id
 
+logger = logging.getLogger(__name__)
```

Comment:
The logger is still here despite the commit message saying it’s getting rid of it. What about removing it?

Reply:
I was wrong: The logger is still used - and it's still apparently the right mechanism for errors in sphinx domains ... (checked current implementation of the Python domain for reference). So I'll just remove the diff.